From d6f716d1730da630967046087f5b1e9120b63873 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 1 Nov 2019 18:57:05 +0800
Subject: [PATCH] ARM: rockchip_defconfig: disable CONFIG_AUDITSYSCALL

CONFIG_AUDITSYSCALL are enabled by default. This causes the
audit of all system call which impacts performance.
Disable these unused configs to improve performance.

Change-Id: I6487ab4034bd48a183107044be9c127b4bd435b6
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 2847c9e73723..fec08d10f6ab 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -1,6 +1,7 @@
 # CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_KERNEL_LZO=y
 CONFIG_AUDIT=y
+# CONFIG_AUDITSYSCALL is not set
 CONFIG_NO_HZ=y
 CONFIG_HIGH_RES_TIMERS=y
 CONFIG_PREEMPT=y
-- 
2.35.3

